src/workerd/jsg/resource.h

```diff
@@ -679,6 +679,9 @@ struct ResourceTypeBuilder {
     inspectProperties = v8::ObjectTemplate::New(isolate);
     prototype->Set(symbol, inspectProperties, static_cast<v8::PropertyAttribute>(
       v8::PropertyAttribute::ReadOnly | v8::PropertyAttribute::DontEnum));
+
+    auto toStringTagSymbol = v8::Symbol::GetToStringTag(isolate);
```

Comment:
As discussed separately on a call, there's a slight risk that adding this property automatically to all `jsg::Object`s *could* be breaking, at least in theory.  If there is code out there that checks for the presence of the `toStringTag` currently and takes action if it is not there, this can break that. I think the risk is probably really quite low but it's non-zero. Do we need a compatibility flag? Interested in what others think @irvinebroque @kentonv @harrishancock 

Specifically, this changes the current default result of calling `obj[Symbol.toStringTag]` on any of our API objects.

Reply:
I think the consensus is that the risk is acceptable, and we are free to make the change without a compatibility flag.